src/GNNGraphs/transform.jl

```diff
@@ -601,32 +606,40 @@ function Flux.batch(gs::AbstractVector{<:GNNHeteroGraph})
         # # [ei[1] for (ii, ei) in enumerate(edge_indices[edge_t])]
         s = cat_features([ei[1] .+ nodesum[src_t][ii] for (ii, ei) in enumerate(edge_indices[edge_t])])
         t = cat_features([ei[2] .+ nodesum[dst_t][ii] for (ii, ei) in enumerate(edge_indices[edge_t])])
```

Comment:
```suggestion
        s = cat_features([ei[1] .+ nodesum[src_t][ii] for (ii, ei) in enumerate(edge_indices[edge_t]) if ei !== nothing])
        t = cat_features([ei[2] .+ nodesum[dst_t][ii] for (ii, ei) in enumerate(edge_indices[edge_t]) if ei !== nothing])
```
It seems to me that the previous version was wrong since `edge_idices[edge_t]` could be of length lengths less than `length(nodesum[src_t])`